src/js/palindrome.js

```diff
@@ -0,0 +1,11 @@
+var isPalindrome = function (str) {
+  var str = document.getElementById("palin").value;
+
+  if (str === str.split('').reverse().join('') || str.length === 1) {
```

Comment:
check for `nitin` and `nitin  ` and see how it behave. Ideally both are palindrome

Reply:
instead of this regex `/[\W_]/g` you can just use 
https://www.w3schools.com/jsref/jsref_trim_string.asp
And `toLowerCase` is good